service/azureconfig/v1/resource_set.go

```diff
@@ -3,14 +3,18 @@ package v1
 import (
 	"fmt"
 
+	"github.com/cenkalti/backoff"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/kubernetes"
+
 	"github.com/giantswarm/azure-operator/client"
 	"github.com/giantswarm/certs"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/framework/resource/metricsresource"
+	"github.com/giantswarm/operatorkit/framework/resource/retryresource"
 	"github.com/giantswarm/randomkeys"
-	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
-	"k8s.io/client-go/kubernetes"
```

Comment:
I was a bit quick with the approve ;)

If this can't be done with tooling I'd prefer to keep the 3 blocks we have now.

Reply:
OK good point! :)

I still prefer 3 blocks because adding the 4th does improve readability much for me. But if others want to go with it that's OK with me.